presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveRepeatFunctionQueries.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.nativeworker;
+
+import org.testng.annotations.Test;
+
+public class TestHiveRepeatFunctionQueries
```

Comment:
I don't think it will scale to add a separate test file for each function. How about renaming to TestHiveArrayFunctionQueries and documenting that this test class is for testing array functions?

Reply:
I think the `TestHiveArrayFunctionQueries` explains itself pretty well and I don't see similar documents in other tests. Is there specific comment we need to add?

Reply:
I think it would be nice to add a class-level comment "Tests for array functions." The class name is fairly descriptive though, hence, the comment is not strictly necessary.

Reply:
Got it. How about let's keep it as it is to be consistent with other tests in the same folder.